### PR TITLE
Docs review

### PR DESCRIPTION
**Pull request by jpalac**
Part of https://github.com/Particular/CustomerSuccess/issues/4201

### CODE REVIEW

File: servicepulse/intro-pending-retries.md [hunk 1]
```diff
@@ -54,9 +54,7 @@ Results can be filtered by queue name using the search functionality:
 
 ![Queue Filter](images/pending-retries-filter-queues.png 'width=500')
 
-Detailed information about the message, such as failure timestamp, endpoint, stack trace of the error, etc., is displayed in the same manner as on the [Failed Messages](intro-failed-messages.md) page providing additional information as follows:
-
- * **Redirect** Information if a redirect is created for this queue.
+Detailed information about the message, such as failure timestamp, endpoint, stack trace of the error, etc., is displayed in the same manner as on the [Failed Messages](intro-failed-messages.md) page providing additional informtaion about [redirects](/servicepulse/redirect.md) if one is created for this queue.
```
Comment (johnsimons) [inline servicepulse/intro-pending-retries.md:57]:
```suggestion
Detailed information about the message, such as the failure timestamp, endpoint, stack trace of the error, etc., is displayed in the same manner as on the [Failed Messages](intro-failed-messages.md) page, which provides additional information about [redirects](/servicepulse/redirect.md) if one is created for this queue.
```

**Review by johnsimons — APPROVED** [review]
Just a small spelling mistake

File: servicepulse/intro-pending-retries.md (hunk 1 above)
Comment (jpalac) [reply servicepulse/intro-pending-retries.md:57]: @johnsimons your suggestion implies the that additional information is available on the FailedMessages page, but that is not the case.

```suggestion
Detailed information about the message, such as the failure timestamp, endpoint, stack trace of the error, etc., is displayed in the same manner as on the [Failed Messages](intro-failed-messages.md) page, additionally providing information about [redirects](/servicepulse/redirect.md) if one is created for this queue.
```

**Review by saratry — COMMENTED** [review]
LGTM

**Pull request merged**